common/define-grammar.js

```diff
@@ -93,25 +108,15 @@ module.exports = function defineGrammar(dialect) {
         optional($._initializer)
       ),
 
-      call_expression: ($, previous) => choice(
-        prec(PREC.CALL, seq(
-          field('function', $._expression),
-          field('type_arguments', optional($.type_arguments)),
-          field('arguments', choice($.arguments, $.template_string))
-        )),
-        prec(PREC.MEMBER, seq(
-          field('function', $._primary_expression),
-          '?.',
-          field('type_arguments', optional($.type_arguments)),
-          field('arguments', $.arguments)
-        ))
+      _call_expression: $ => choice(
+        callExpression($)(PREC.CALL),
+        callExpression($)(PREC.MEMBER, '?.'),
       ),
+      call_expression: $ => $._call_expression,
```

Comment:
Looks like `_call_expression` could be inlined here. It's this way probably because I was trying to reuse it in `new_expression`.

Reply:
solved